spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -159,7 +160,7 @@ public void testBinPackUnpartitionedTable() {
 
   @Test
   public void testBinPackPartitionedTable() {
-    Table table = createTablePartitioned(4, 2);
+    Table table = createTablePartitioned(4, 2, new HashMap<>());
```

Comment:
Adding the additional message signature will remove the need to change these lines

Reply:
ok.